core/src/main/python/wlsdeploy/tool/discover/deployments_discoverer.py

```diff
@@ -325,6 +338,54 @@ def _create_app_folder(self, application_name, application_dict):
 
         _logger.exiting(class_name=_class_name, method_name=_method_name)
 
+    def _jdbc_password_fix(self, source_name):
+        """
+        This will look for password and userid in the jdbc standalone xml and
+        replace with either fix password token or a token in the xml and variable file.
+        It extracts the jdbc xml from the archive and then replaces it with the updated file.
+        :param source_name: Name of the path and file for the standalone xml file
+        """
+        _method_name = '_jdbc_password_fix'
+        _logger.entering(source_name, class_name=_class_name, method_name=_method_name)
+        archive_file = self._model_context.get_archive_file()
+        tmpDir = FileUtils.getTmpDir();
+        temp_file = FileUtils.createTempDirectory(tmpDir, 'jdbc-xml')
+        jdbc_file = archive_file.extractFile(source_name, temp_file)
+        jdbc_out = FileUtils.createTempDirectory(tmpDir, 'jdbc-out')
+        jdbc_out = archive_file.extractFile(source_name, jdbc_out)
+        bis = BufferedReader(FileReader(jdbc_file))
+        bos = BufferedWriter(FileWriter(jdbc_out))
+
+        found = False
+        while bis.ready():
+            line = bis.readLine()
+            if 'password-encrypted' in line:
```

Comment:
This assumes there is no text in the file written as `password-encrypted`.  Parsing the xml file should avoid this type of assumption.

Reply:
Also, there is another password encrypted in JDBCOracleParams..

Reply:
This is reading each line, simple regexp expression to find the line that has password-encrypted. Its not parsing the file, is that what you are suggesting?

Reply:
We already discussed offline,  in the unlikely event that someone put a  note that has `password-encrypted`  then it will fail.  The xml should be able to parse by jaxb or like api, it takes a little more time but is less brittle.  

Anther way is to look for `<password-encrypted>**</password-encrypted>`  with regex.